texturepacker/pluginsrc/com/defold/bob/pipeline/tp/Atlas.java

```diff
@@ -137,10 +137,14 @@ static public TextureImage createTexture(String path, boolean isPaged, List<Buff
     // Used from editor
     // returns an array of floats (flattened (x,y)-tuples): [x0,y0,x1,y1,x2,...]
     static public float[] getTriangles(TextureSetLayout.SourceImage image, Float pageHeight) {
-        float half_width = image.rect.width * 0.5f;
-        float half_height = image.rect.height * 0.5f;
-        float centerx = image.rect.x + half_width;
-        float centery = image.rect.y + half_height;
+
+        // The image may be rotated, then so is the rectangle
+        float originalWidth = image.rotated ? image.rect.height : image.rect.width;
+        float originalHeight = image.rotated ? image.rect.width : image.rect.height;
```

Comment:
We need the original size when using it with the vertices (as they're not rotated)